### PR TITLE
Feat/age checkbox

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
ClickUp Ticket: https://app.clickup.com/t/8669f4rzx

What (if any) features are you implementing?
- Add "I confirm I am 13 years of age or older." to `step-1`.
- I refactored the media query for the the `.logo` and `.jeffcoLogo` in order to resolve the overflow of the `jeffcoLogo`:

Visual overflow:
Screenshot 2023-08-31 at 1 30 03 PM

Media query fix:
Screenshot 2023-08-31 at 1 34 52 PM

Initial state:
Screenshot 2023-08-31 at 1 20 42 PM

Falsy state:
Screenshot 2023-08-31 at 1 17 45 PM

Truthy state:
Screenshot 2023-08-31 at 1 18 01 PM

Mobile:
Screenshot 2023-08-31 at 1 18 25 PM

### CODE REVIEW

File: src/Components/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -46,13 +47,18 @@ const LandingPage = ({ setFetchedScreen, handleCheckboxChange }: LandingPageProp
   });
 
   useEffect(() => {
-    errorController.updateError(formData.agreeToTermsOfService);
-  }, [formData.agreeToTermsOfService]);
+    privacyErrorController.updateError(formData.agreeToTermsOfService);
+    ageErrorController.updateError(formData.is13OrOlder);
+  }, [formData.agreeToTermsOfService, formData.is13OrOlder]);
```
Comment (CalebPena) [inline src/Components/LandingPage/LandingPage.tsx:52]: Can you split this into separate useEffects?

File: src/Components/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -102,6 +108,10 @@ const LandingPage = ({ setFetchedScreen, handleCheckboxChange }: LandingPageProp
     );
   };
 
+  const createAgeCheckboxLabel = () => {
+    return <FormattedMessage id="disclaimer-label-age" defaultMessage="I confirm I am 13 years of age or older." />;
+  };
```
Comment (CalebPena) [inline src/Components/LandingPage/LandingPage.tsx:113]: I don't think it is necessary to have a function for this. I think we could just put the FormattedMessage where this function is called.

File: src/Components/Wrapper/Wrapper.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ const initialFormData: FormData = {
   isTest: undefined,
   externalID: undefined,
   agreeToTermsOfService: false,
+  is13OrOlder: false,
```
Comment (CalebPena) [inline src/Components/Wrapper/Wrapper.tsx:15]: This needs to be posted and fetched from the backend.

**Review by CalebPena — CHANGES_REQUESTED** [review]

File: src/Components/LandingPage/LandingPage.tsx (hunk 1 above)
Comment (ivonne-hernandez) [reply src/Components/LandingPage/LandingPage.tsx]: Absolutely! Good call.

**Pull request merged**